loopd/swapclient_server.go

```diff
@@ -752,13 +752,20 @@ func (s *swapClientServer) GetLiquidityParams(_ context.Context,
 func newRPCRule(channelID uint64, peer []byte,
 	rule *liquidity.TypedRule) *looprpc.LiquidityRule {
 
-	return &looprpc.LiquidityRule{
+	rpcRule := &looprpc.LiquidityRule{
 		ChannelId:         channelID,
 		Pubkey:            peer,
 		Type:              looprpc.LiquidityRuleType_THRESHOLD,
 		IncomingThreshold: uint32(rule.MinimumIncoming),
 		OutgoingThreshold: uint32(rule.MinimumOutgoing),
+		SwapType:          looprpc.SwapType_LOOP_OUT,
 	}
+
+	if rule.Type == swap.TypeIn {
```

Comment:
Would be nice (why oh why do they not match 😭 ), but I think both need migrations? Either db migration or rpc enum (since rpc relies on numerical value), I think it's too messy to justify the small win.